compiler/include/visibleFunctions.h

```diff
@@ -50,7 +50,7 @@ class VisibilityInfo {
   bool inPOI() { return poiDepth > 0; }
 };
 
-bool isTypeHelperName(const char* fnName);
+bool isInternalTypeHelperName(const char* fnName);
```

Comment:
Not a big deal but this does not appear to be called outside of visibleFunctions.cpp so might be better as a static function?